app/models/interaction.rb

```diff
@@ -4,4 +4,5 @@ class Interaction < ApplicationRecord
   belongs_to :medium
 
   has_many :notes, dependent: :destroy
+  accepts_nested_attributes_for :notes, allow_destroy: true
```

Comment:
@santiagosan93 does not like nested attributes. I also used them in my PR, but let's talk about this in the future. Why does Santi not like them, and what can we use instead? 

<img width="698" alt="Screenshot 2024-04-05 at 12 27 10" src="https://github.com/emmvs/call_your_mom/assets/90188399/9a2524de-397a-4373-adda-1cfc81562f48">

Reply:
or this?

Reply:
I'll look into it but I guess we discuss, Santiago Sanchess will know something